client/signup/steps/domains-pm/utils.js

```diff
@@ -0,0 +1,30 @@
+import validUrl from 'valid-url';
```

Comment:
These are unlikely to change I think, even if the steps change visually. Can we reference the existing files (with the same content)? As now we're duplicating the utils and their corresponding test files, with the same content.